data-structures/ds-stk/java/Stack.java

```diff
@@ -0,0 +1,80 @@
+class Stack {
+    // store elements of stack
+    private int arr[];
+    // represent top of stack
+    private int top;
+    // total capacity of the stack
+    private int capacity;
+    // construct a stack
+    Stack(int size) {
+      arr = new int[size];
+      capacity = size;
+      top = -1;
+    }
+
+    // push elements to the top of stack
+    public void push(int x) {
+      if (isFull()) {
+        System.out.println("Stack OverFlow");
+  
+        // terminates the program
+        System.exit(1);
```

Comment:
Codacy found an issue: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6332826)